public/assets/js/validateOtp.js

```diff
@@ -0,0 +1,35 @@
+document.addEventListener("DOMContentLoaded", function (event) {
```

Comment:
Why use this instead of jQuery?

Reply:
Sir this function has no use now so I delete it out.